notary_now_api/views.py

```diff
@@ -42,27 +42,32 @@ def notary_verify(request, pk):
         return JsonResponse({'error': 'Notary not Found'}, status=404)
 
 @csrf_exempt
-def make_appointment(request, pk):
+def appointments(request, pk):
     notary = Notary.objects.filter(user_id=pk)[0]
     if notary:
-        appointment_info = json.loads(request.body)
-        appointment = Appointment.objects.filter(
-            notary_id=pk,
-            appointee_id=appointment_info['appointee_id'],
-            time=appointment_info['time'],
-            date=appointment_info['date'],
-            location=appointment_info['location'],
-        )
-        if not appointment:
-            Appointment.objects.create(
+        if request.method == 'GET':
```

Comment:
Great job doing the switch between methods

Reply:
Great work! I am so happy to have you all on this team!